tests/output/test_iac_text_output.py

```diff
@@ -25,11 +26,26 @@ def test_display_single_vulnerabilities(cli_fs_runner: CliRunner):
             "tmp",
         ],
     )
-
     assert_iac_version_displayed(result)
     assert_file_single_vulnerability_displayed(result)
 
 
+@my_vcr.use_cassette("test_iac_scan_single_vulnerability")
+def test_exit_zero_single_vulnerability(cli_fs_runner: CliRunner):
+    Path("tmp/").mkdir(exist_ok=True)
```

Comment:
You can use pytest `tmp_path` fixture here (https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture), to avoid leaving temporary directories behind.